db/sg_replicate_cfg.go

```diff
@@ -434,6 +468,12 @@ func (m *sgReplicateManager) InitializeReplication(config *ReplicationCfg) (repl
 		return nil, err
 	}
 
+	// Explicitly defined auth credentials of remote database takes precedence over
+	// the credentials specified in the database URL.
+	if config.Username != "" && config.Password != "" {
+		rc.PassiveDBURL.User = url.UserPassword(config.Username, config.Password)
+	}
```

Comment:
Unlikely, but password _could_ be empty in this case. Think we just want to check for username.

Reply:
Sure, let's remove the check for password.